build/containers/marianas/0.0.0/Singularity

```diff
@@ -0,0 +1,45 @@
+Bootstrap: docker
+From: /pipeline-marianas:0.0.0
+Registry: http://localhost:5000
+
+%setup
+
+    # copy settings-container.sh from the host to the container
+    cp /vagrant/build/scripts/settings-container.sh $SINGULARITY_ROOTFS/tmp
+
+%post
+
+    # load the settings-container.sh which was copied in the %setup step
+    source /tmp/settings-container.sh
+
+    # create an empty directory for each bind point defined
+    for dir in $SINGULARITY_BIND_POINTS
+    do
+        mkdir -p $dir
+    done
+
+    # remove settings-container.sh
+    rm -rf /tmp/settings-container.sh
+
+%runscript
+
+    # just run java so that caller can pass java options
+    # exec java -jar /usr/bin/gatk.jar "$@"
+    exec java "$@"
+
+# %test
+#
+#     # get actual output of the tool
+#     exec /usr/bin/marianas-0.0.0.jar "$@"
+#
+#     # expected output
+# cat > /tmp/expected.diff.txt << EOM
+# Program: Marianas
+# Version: 0.0.0
+# EOM
+#
+#     # diff
+#     diff /tmp/actual.diff.txt /tmp/expected.diff.txt
+#
+#     # delete tmp
+#     rm -rf /tmp/*.diff.txt
```

Comment:
I would like to include a test for this actually, but there's no good default behavior for when not passing correct params to this jarfile. Perhaps we could include some test fastqs and have the test run off of those.

Reply:
√